src/pages/VolunteersJobListingFS.page

```diff
@@ -159,6 +168,7 @@
                     	if (x == '') x = JobShiftDateTimeString('{!shift.Start_Date_Time__c}', '{!shift.Duration__c}');
                         OpenSignUpDlg(event.clientX, event.clientY, '{!job.Id}', '{!shift.Id}', '{!JSENCODE(job.Name)}', x); 
                         return false;"
+                        style="{!IF(job.External_Signup_Url__c == null, '', 'display: none;')}"
```

Comment:
@davidhabib I have just moved the display logic from the style to the parent panel rendered attribute, please let me know if you notice anything else. Thank you.